bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/testcases/MainActivityTest.kt

```diff
@@ -0,0 +1,117 @@
+package net.gini.android.bank.sdk.exampleapp.ui.testcases
+
+import android.Manifest
+import android.app.Activity
+import android.app.Instrumentation
+import android.content.Intent
+import android.net.Uri
+import androidx.test.espresso.intent.Intents
+import androidx.test.espresso.intent.Intents.intending
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasAction
+import androidx.test.espresso.intent.matcher.IntentMatchers.hasType
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.filters.LargeTest
+import androidx.test.rule.GrantPermissionRule
+import net.gini.android.bank.sdk.exampleapp.ui.MainActivity
+import net.gini.android.bank.sdk.exampleapp.ui.screens.CaptureScreen
+import net.gini.android.bank.sdk.exampleapp.ui.screens.HelpScreen
+import net.gini.android.bank.sdk.exampleapp.ui.screens.MainScreen
+import net.gini.android.bank.sdk.exampleapp.ui.screens.OnboardingScreen
+import org.hamcrest.Matchers.allOf
+import org.junit.After
+import org.junit.Before
+import org.junit.FixMethodOrder
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.MethodSorters
+
+
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
```

Comment:
[Request] Tests should be independent and we should not have a fixed method order. See my other comment where I tried to explain why.

Reply:
Hmm, i understand that a test should run independent but if I remove the `@FixMethodOrder`, then there is no specific defined order in which tests should run and this keep on failing the test as with every launch the onboarding screens appears.. 🥲 
<img width="753" alt="image" src="https://github.com/gini/gini-mobile-android/assets/144695421/a1d143ee-c46f-47ca-8e98-11878eb38f3a">

Reply:
To avoid having tests run in a set order you need to perform the necessary steps in each test's _setup_ section to bring the SDK into the state where it fullfills the preconditions. Check [here](https://github.com/gini/gini-mobile-android/pull/462#discussion_r1670804554) the setup section in my code example for the `skipsOnboardinAndShowsCameraScreenOnSubsequentLaunches()` test.